components/f4jumble/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "f4jumble"
+description = "Implementation of Zcash's F4Jumble algorithm"
+version = "0.0.0"
+authors = [
+    "Jack Grigg <jack@electriccoin.co>",
+    "Kris Nuttycombe <kris@electriccoin.co>",
```

Comment:
```suggestion
    "Kris Nuttycombe <kris@electriccoin.co>",
    "Daira Hopwood <daira@electriccoin.co>",
```

Reply:
Huh, I looked in the code history and thought I only saw Kris and myself as contributors to the implementation. Apologies for missing you!